switch/crossfile.sh

```diff
@@ -18,7 +18,8 @@ function bin_path() {
 	which "${TOOL_PREFIX}$1"
 }
 
-ADDITIONAL_LINK_FLAGS="-specs=$DEVKITPRO/libnx/switch.specs"
+# Link with libstdc++ since libEGL requires it
+ADDITIONAL_LINK_FLAGS="-specs=$DEVKITPRO/libnx/switch.specs -lstdc++"
```

Comment:
Is this really necessary? I'm pretty sure that since #240 Taisei already depends on a C++ standard library, because the basis transcoder is C++ and uses some STL stuff. Meson should automatically use a C++ compiler (e.g. `g++`) at the linking stage because of this, which I think already should pull in the C++ stdlib. Does that not work as is?

Reply:
You're right it's not needed since the texture PR, this was done prior, and I removed it